### DISCUSSION TITLE
forwardMessage Image is possible?

### DISCUSSION THREAD

**Discussion by doprado** [3mora2/WPP_Whatsapp#53] [General]
I understand that "image" messages cannot be forwarded. However, "chat" messages can be forwarded. Is this a limitation or a non-implementation issue?

I usually send promotional messages where I post an image and the description of the promotion in the caption.

**Comment by 3mora2** [comment]
I dont konw, open issue in [wa-js](https://github.com/wppconnect-team/wa-js)
